libV2/server/middleware.js

```diff
@@ -80,11 +81,40 @@ async function initializeOasTools(spec, app) {
     return promisify(oasTools.initialize)(spec, app);
 }
 
+// eslint-disable-next-line no-unused-vars
+async function authV4Middleware(request, response, params) {
+    const authHeader = request.headers.authorization;
+    if (!authHeader || !authHeader.startsWith('AWS4')) {
```

Comment:
could be a minor point, but I think it's more correct to stop comparing after the delimiter (space?) than before, so like `startsWith("AWS4 ")`

Reply:
I'm not very knowledgeable about what we support wrt authV4, but I was expecting a header in the format `AWS4-HMAC-SHA256`. I don't know if we support other signing algos so it may make sense to compare the entire string? The logic here was mostly lifted from `/lib/Vault.js` and `router/Router.js`

Reply:
Yes why not comparing against `AWS-HMAC-SHA256[delim]` then, but keeping what is already there in vault code and works is fine too.

Reply:
I've added a `-`